show-grid/src/main/res/layout/activity_main.xml

```diff
@@ -1,27 +1,31 @@
 <?xml version="1.0" encoding="utf-8"?>
-<layout xmlns:android="http://schemas.android.com/apk/res/android"
+<layout xmlns:tools="http://schemas.android.com/tools"
+    xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto">
 
-    <androidx.constraintlayout.widget.ConstraintLayout xmlns:tools="http://schemas.android.com/tools"
+    <androidx.constraintlayout.widget.ConstraintLayout
         android:layout_width="match_parent"
         android:layout_height="match_parent"
         tools:context=".MainActivity">
 
         <com.arcgismaps.mapping.view.MapView
             android:id="@+id/mapView"
             android:layout_width="match_parent"
-            android:layout_height="match_parent" />
+            android:layout_height="0dp"
+            app:layout_constraintBottom_toTopOf="@id/menuButton"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent" />
 
         <com.google.android.material.button.MaterialButton
             android:id="@+id/menuButton"
+            style="@style/OutlineButton"
```

Comment:
Question: why did we choose this to be an Outline button and not a regular button?
It seems like a primary/main button, as the intension of this sample is to show different grids( having different label color and positions) which happens on click of this button.

Reply:
The guideline on emphasis was irrelevant to me when it came to just having one button. It makes sense for a button to be "emphasized" for a layout with two or more buttons. If a layout consists of only one button, the style used could just be arbitrary, seen as in the user has no other UI interaction. 

My choice for a outlined button with singular button layout was pretty much arbitrary, although I felt it helped bring down the (for lack of a better word) purpleness of the layout in light mode. 

Either option would still respect the material theme guidelines, so it's up to us to decide. Would you prefer keeping it as an outline button or choose to make contained buttons as the default of singular button layouts? 

<img width="300" alt="image" src="https://github.com/Esri/arcgis-maps-sdk-kotlin-samples/assets/14892474/741469b7-e98e-426d-8808-a0ca93a705ce">

<img width="300" alt="image" src="https://github.com/Esri/arcgis-maps-sdk-kotlin-samples/assets/14892474/2c0e8f62-985d-4cfd-8de5-0df7eb17477f)">

Reply:
Sounds good, I've removed the outline from the following samples:
- Display dimensions
- Download vector tiles
- Show Grid 

I've updated the guideline to follow this suggestion as well. 👍